frontend/packages/dev-console/src/components/pipelineruns/detail-page-tabs/PipelineRunDetails.tsx

```diff
@@ -19,16 +19,45 @@ export const PipelineRunDetails: React.FC<PipelineRunDetailsProps> = ({ obj: pip
           <ResourceSummary resource={pipelineRun} />
         </div>
         <div className="col-sm-6">
+          <SectionHeading text="Pipeline" />
           <dl>
-            <dt>Pipeline</dt>
             <dd>
-              <ResourceLink
-                kind={referenceForModel(PipelineModel)}
-                name={pipelineRun.spec.pipelineRef.name}
-                namespace={pipelineRun.metadata.namespace}
-              />
+              <React.Fragment key={pipelineRun.spec.pipelineRef.name}>
+                <dt>Name: {pipelineRun.spec.pipelineRef.name}</dt>
+                <ResourceLink
+                  kind={referenceForModel(PipelineModel)}
+                  name={pipelineRun.spec.pipelineRef.name}
+                  namespace={pipelineRun.metadata.namespace}
+                  inline
+                />
+              </React.Fragment>
             </dd>
           </dl>
+          <br />
+          {pipelineRun?.spec?.resources.length && (
+            <>
+              <SectionHeading text="Pipeline Resources" />
+              <dl>
+                {pipelineRun?.spec?.resources.map((res) => {
+                  return (
+                    <React.Fragment key={res.resourceRef.name}>
+                      <dd>
+                        <React.Fragment key={res.resourceRef.name}>
+                          <dt>Name: {res.resourceRef.name}</dt>
```

Comment:
```
<>
  <dt>...</dt>
  <dd>...</dd>
<>
```
`dt` and `dd` are siblings in the [DOM spec](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dd).

`dt` = Description Term
`dd` = Description Detail

Your name is the term, your resource link is the detail.

Reply:
Updated it.